README.md

```diff
@@ -1,23 +1,25 @@
 [![Build Status](https://travis-ci.org/GoogleCloudPlatform/forseti-security.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/forseti-security) [![Coverage Status](https://coveralls.io/repos/github/GoogleCloudPlatform/forseti-security/badge.svg)](https://coveralls.io/github/GoogleCloudPlatform/forseti-security)
 
 # Forseti Security
-Forseti Security helps you secure your Google Cloud Platform (GCP) environment.
+Forseti Security is a tool that secures your organization's use of Google Cloud
+Platform.
 
-Features:
+For details on installation see
```

Comment:
nit: instead of just mentioning installation, how about phrasing it as "installation and documentation"?